exampleCourse/courseInstances/SectionA/assessments/16-gallery-drawing-PL-elements/infoAssessment.json

```diff
@@ -1,27 +1,15 @@
 {
-    "uuid": "7A6CFC5A-75AF-49E8-BFBD-035E21EE4F8E",
+    "uuid": "6338DE4E-7E66-4B58-8CB4-1E126FB9AF9C",
```

Comment:
very much a nit: we're not very consistent between upper+lowercase UUIDs. I think we should stick to lowercase.

Reply:
Going to ignore this for now.